### PR TITLE
Fix issues in DALI in action snippet

### PR DESCRIPTION
**Pull request by jantonguirao**
Signed-off-by: Joaquin Anton <janton@nvidia.com>

## Category:

**Other** (documentation fix)

## Description:

- Fixes issues reported by https://github.com/NVIDIA/DALI/issues/4267
- Misuse of single quotes
- Runable snippet
- Definition of data source
- shuffle -> random_shuffle

### Affected modules and functionalities:

Documentation

### Key points relevant for the review:

Check render documentation

### Tests:

- [ ] Existing tests apply
- [ ] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [x] N/A

### Documentation
- [ ] Existing documentation applies
- [x] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [x] RST
  - [ ] Jupyter
  - [ ] Other
- [ ] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [x] N/A

**REQ IDs**: N/A

**JIRA TASK**: N/A

### CODE REVIEW

**Comment by jantonguirao** [comment]
!build

**Comment by jantonguirao** [comment]
!build

**Review by stiepan — APPROVED** [review]

File: README.rst [hunk 1]
```diff
@@ -37,32 +37,56 @@ DALI in action::
...
+          mean=[0.485 * 255, 0.456 * 255, 0.406 * 255],
+          std=[0.229 * 255, 0.224 * 255, 0.225 * 255],
+          mirror=fn.random.coin_flip())
+      return images, labels
+  
+  
   train_data = DALIGenericIterator(
-     [get_dali_pipeline(data_dir, (244,244))], ['data', 'label’],
-     reader_name='Reader’
+      [get_dali_pipeline(batch_size=16)], ['data', 'label'],
```
Comment (mzient) [inline README.rst:80]: Nitpick
```suggestion
     [get_dali_pipeline(batch_size=16)], ['data', 'label'],
```

Comment (mzient) [inline README.rst:80]: Readability - I thought that the entire line was one array, when in fact there are two.
```suggestion
      [get_dali_pipeline(batch_size=16)],
      ['data', 'label'],
```

**Review by mzient — APPROVED** [review]
...with a nitpick.

**Comment by jantonguirao** [comment]
!build

**Pull request merged**